src/containers/Login/components/AddProfileDetails.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import UserSettings from 'containers/Settings/UserSettings';
+import { Modal, Text, Button } from 'components';
+import { PageCard } from 'explorer-components';
+import { actions } from '../reducer';
+
+const AddProfileDetails = props => {
+  const { visible, onClose, showForm, form } = props;
+
+  if (form) {
+    return (
+      <Modal visible={visible} size="large" dismissable onClose={onClose}>
+        <Modal.Header closable icon={['fal', 'faAddressCard']}>
```

Comment:
This isn't showing up for me - showing up for you? The Icon.

Reply:
The icon is also not showing up for me, I wanted to ask you if you might know why? I thought I had imported everything correctly, but obviously not.